yaml/generator.go

```diff
@@ -78,6 +78,16 @@ func (c *TestCase) generateDockerComposeFile() []byte {
 	vars["LokiHTTPPort"] = c.PortConfig.LokiHTTPPort
 	vars["TempoHTTPPort"] = c.PortConfig.TempoHTTPPort
 
+	env := []string{}
+
+	for k, v := range vars {
+		env = append(env, "OATS_"+strings.ToUpper(k)+"="+v.(string))
```

Comment:
```suggestion
		env = append(env, k+"="+v.(string))
```

Reply:
prepend OATS_ and make upper case in all usages - that makes it easier to follow